client/state/ui/reducer.js

```diff
@@ -23,6 +23,7 @@ import gutenbergOptInDialog from './gutenberg-opt-in-dialog/reducer';
 import language from './language/reducer';
 import layoutFocus from './layout-focus/reducer';
 import mediaModal from './media-modal/reducer';
+import mySites from './my-sites/reducer';
```

Comment:
we should really pull this in only for routes that load my-sites. @jsnajdr can you help with that?